app/assets/javascripts/discourse/app/lib/search.js

```diff
@@ -244,3 +244,14 @@ export function updateRecentSearches(currentUser, term) {
   recentSearches.unshiftObject(term);
   currentUser.set("recent_searches", recentSearches);
 }
+
+export function logSearchLinkClick(params) {
+  ajax("/search/click", {
```

Comment:
Do we need to handle errors here? I'm actually not sure what happens if an error is thrown here and we don't handle it. I think it appears in our logging?